### ISSUE TITLE
Amplify CLI doesn't allow selection of app type when pulling studio project

### ISSUE THREAD

**Issue by adplant** [aws-amplify/amplify-cli#10897]
Versions:

Amplify: 9.2.1
Node: V16.16.0
NPM: 8.11.0

I am trying to pull a project by using:

`amplify pull --appId ############ --envName dev`

Things appear to work until I get to the section where I am supposed to select the **app type**. When that option comes up, JavaScript is already highlighted, and I am unable to use the arrow keys to go up and select anything else (in my case, I want to select Flutter).

```
Confirm login in the browser or manually paste in your CLI login key:
√ Successfully received Amplify Studio tokens.
Amplify AppID found: ###############. Amplify App name is: #############
Backend environment dev found in Amplify Console app: 
? Choose your default editor: Android Studio
? Choose the type of app that you're building (Use arrow keys)
  android
  flutter
  ios
> javascript
```

I'm using a windows 10 machine.

I've tried doing as a regular user and as an admin on the Command Prompt and PowerShell. The same results for both.

I tried just doing a regular

`amplify init`

And the problem doesn NOT exist when I do that. I am able to arrow up and down without a problem.

This seems like it has been a problem in the past, but was supposedly fixed. I don't know if it's an issue with the most current Amplify CLI version or what.

If this is a bug, can someone tell me how to at least do a work around until it's fixed.

Thanks.

Follow up....I did confirm with a colleague that this issue doesn't happen on her Mac in the standard terminal.

**Comment by adplant** [comment]
FYI, a workaround I've found is to use the Git Bash for Windows terminal and do a blind arrow selection. It's not ideal, but it works.
Ex. when the javascript option is the option with the arrow, I just hit the up arrow twice and hit 'enter' to get Flutter.
I actually had to do this for the question prior (default editor) as well when I used Git Bash for Windows. But like I said, it seems to have worked.

**Comment by ykethan** [comment]
Hello @adplant, I was able to reproduce the issue. Marking as bug.

Note: The issue occurs when performing a pull on studio enable backend. Tested on a backend without studio enabled the issue does not occur.

**Comment by ykethan** [comment]
Note: wsl on windows works as expected as well.
Powershell does not work

**Comment by ykethan** [comment]
Note for the team: was able to reproduce the issue in Amplify CLI v11.0.0.

**Comment by lazpavel** [comment]
As a workaround, pressing the enter key once will unfreeze the arrow keys

**Comment by swaminator** [comment]
Fix for this will go out with 11.0.4
